# Hard-Ass Python 

Here we will do everything we can to code as hell. 

Yes, with our hands until we have a solid grasp on Python.


## Variables and Names 

In [1]:
cars = 100
space_in_car = 4.0
drivers = 30
passengers = 100
cars_not_driving = cars - drivers
cars_driving = drivers
carpool_capacity = cars_driving*space_in_car
avg_passengers_per_car = passengers / cars_driving


In [5]:
# Now lets print our logic we just came up with...

print("There are", cars, "cars available to you!")
print("There are only", drivers, "drivers available.")
print("There will be", cars_not_driving, "empty cars today.")
print("We can take", carpool_capacity, "people today")
print("Today we have", passengers, "to carpool!")
print("We need to put approximately", avg_passengers_per_car, "in each car.")

There are 100 cars available to you!
There are only 30 drivers available.
There will be 70 empty cars today.
We can take 120.0 people today
Today we have 100 to carpool!
We need to put approximately 3.3333333333333335 in each car.


##### Wait a minute, how can you put 3.33 people in each car

Exercise 1 : Round this number off, will you!